.evergreen/scripts/resync-all-specs.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import argparse
+import os
+import pathlib
+import subprocess
+
+
+def resync_specs(directory: pathlib.Path, errored: dict[str, str]) -> list[str]:
+    """Actually sync the specs"""
+    for spec in os.scandir(directory):
+        if not spec.is_dir():
+            continue
+
+        if spec.name in ["asynchronous"]:
+            continue
+        process = subprocess.run(
+            ["bash", "./.evergreen/resync-specs.sh", spec.name],  # noqa: S603, S607
+            capture_output=True,
+            text=True,
+            check=False,
+        )
+
+        if process.returncode != 0:
+            errored[spec.name] = process.stderr
+
+    process = subprocess.run(
+        ["git diff --name-only | awk -F'/' '{print $2}' | sort | uniq"],  # noqa: S607
```

Comment:
Note: 
```
S607 Starting a process with a partial executable path
```
I tried to separate the string into `["git", "diff --name-only | awk -F'/' '{print $2}' | sort | uniq"]` but that would cause the command to fail? So I kept it as a whole string..